src/Illuminate/Broadcasting/Broadcasters/RedisBroadcaster.php

```diff
@@ -105,6 +108,8 @@ public function validAuthenticationResponse($request, $result)
      * @param  string  $event
      * @param  array  $payload
      * @return void
+     *
+     * @throws \Illuminate\Broadcasting\BroadcastException
```

Comment:
can't add throws doc here unless it also exists on the contract

Reply:
Used the same logic, there is on the PusherBroadcaster.

Should I add it to the contract?

Reply:
Just added. makes sense.